open-hackathon-server/src/hackathon/views/resources.py

```diff
@@ -266,6 +266,14 @@ def delete(self):
         return team_manager.dismiss_team(g.user, self.context().id)
 
 
+class TeamCheckNameResource(HackathonResource):
+    @hackathon_name_required
+    def get(self):
+        parse = reqparse.RequestParser()
+        parse.add_argument('name', type=str, location='args', required=True)
```

Comment:
We prefer not use the old way to validate argument. Can you create a json file in `views/schema/`?